src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx

```diff
@@ -1,325 +1,205 @@
+});
+```
+</Side>
+</SideBySide> 
+
+<SideBySide>
+    <Side>
+### Search a website [#advanced-search]
+
+Beyond text sweeps, your scripted browser monitor can locate elements via their `class`, `id`, `name`, or XPath. This script:
+
+1. Goes to your site.
+2. Locates a search box via its XPath, then types `relic`.
+3. Locates a submit button via its XPath, then clicks it to submit the search.
+
+    </Side>
+    <Side>
+```js
+$webDriver.get("http://example.com/static/companyBlog.jsp").then(function(){
+//Find the search field by specifying its `id`, then enter `relic`.
+    return $webDriver.findElement($driver.By.xpath("//h4[text()='Blog Search']/following-sibling::div/input")).sendKeys("relic");
+}).then(function(){
+//Click the search button.
+    return $webDriver.findElement($driver.By.xpath("//h4[text()='Blog Search']/following-sibling::div//button")).click();
+});
+```
+    </Side>
+</SideBySide>
+
+<SideBySide>
+    <Side>
+##  Wait for a page element [#wait-for-element] 
+    
+    Your script may need to wait an interval before reporting a failure, since images or dynamic elements can take time to generate on a page. This script:
+
+1. Goes to the example site.
+2. Finds the **Support** dropdown via `HTML ID`, then clicks it.
+3. Waits up to 20 seconds for the **FAQ** button to appear, then clicks it.
+    </Side>
+    <Side>
+```js
+$webDriver.get("http://example.com/").then(function(){
+    return $webDriver.findElement($driver.By.id("supportDropDown")).click();
+}).then(function(){
+    return $webDriver.waitForAndFindElement($driver.By.id("supportFAQLink"), 20000).then(function(aboutPage){
+        return aboutPage.click();
+    })
+});
+```  </Side>
+</SideBySide>
+
+<SideBySide>
+    <Side>
+## Log in to a website [#advanced-login]
+
+You can take your stored secure credentials to test certain page elements on your web apps. For scripting purposes, we recommend that you create new credentials unique to your script. Don't use your personal credentials or reuse credentials when running a monitor. This script:
+
+1. Loads `http://example.com/login.jsp`
+2. Locates the username field by the element name, then submit a secure username [via our secure credentials feature.](/docs/synthetics/new-relic-synthetics/using-monitors/secure-credentials-store-credentials-information-scripted-browsers)
+3. Locates the password field by the element name, then submit a secure password.
+4. Finds the login button via its XPath, then clicks to submit the account information.
+
+For more information about using secure credentials, see [Store secure credentials](/docs/synthetics/new-relic-synthetics/using-monitors/secure-credentials-store-credentials-information-scripted-browsers).
+    </Side>
+    <Side>
+```js
+$webDriver.get("http://example.com/login.jsp").then(function(){
+    return $webDriver.findElement($driver.By.name("username")).sendKeys($secure.SECURE_USERNAME);
```

Comment:
All of these need to use $selenium instead of $driver.

This also needs to be a working example.